site/docs/integrations/docker-extension.md

```diff
@@ -2,4 +2,115 @@
 title: Livecycle Docker Extension
 ---
 
-TBA
\ No newline at end of file
+# Livecycle Docker Extension 🐳
+
+## Overview
+
+Livecycle Docker Extension helps you to share your local development environment with your team. It allows you to share your local development environment with your team, so you can get feedback much earlier in the development workflow, without the hassle of staging environments or CI builds.
+
+## Key Features
+
+- Instant, secure tunneling to frontend and backend services
+- Share HTTPS links so teammates can quickly access and review code while it's in flight
+- Public and private access controls
+- Integration with external identity providers for secure access
```

Comment:
I don't think we currently offer that feature

Reply:
Maybe we need to rephrase it, the external identity is offered by Livecycle.run (with Google/Github and Livecycle Org)

Reply:
Hey, can you tell then what should say here?